src/service/data_products/taxa_count.py

```diff
@@ -208,7 +226,12 @@ async def get_taxa_counts(
             + "Note that if a selection ID is set, any load version override is ignored."),
     status_only: QUERY_VALIDATOR_STATUS_ONLY = False,
     load_ver_override: QUERY_VALIDATOR_LOAD_VERSION_OVERRIDE = None,
-    user: kb_auth.KBaseUser = Depends(_OPT_AUTH)
+    user: kb_auth.KBaseUser = Depends(_OPT_AUTH),
+    sort_priority: list[str] = Query(
+        default=list(_SORT_PRIORITY_ORDER_MAP.keys()),
+        description="Specify the priority order for sorting taxa counts based on the provided fields and "
+                    "their corresponding order.",
```

Comment:
I would put this prior to status_only as it's a control users are likely to use, not just admins or stuff that's only really of interest to UI coders

Reply:
👍